cfgen/cfgen

```diff
@@ -803,8 +803,16 @@ class ConfProcess(ToDhall):
                               services=[])
         m0conf[parent].processes.append(proc_id)
 
+        params = []
+        if proc_t is ProcT.m0_client_other:
+            params = ["origin:in-volatile"]
```

Comment:
It would help to have some brief documentation here for the given params. They seem a bit opaque for a reader as of now. Thank you.

Reply:
Done.